config/environments/development.rb

```diff
@@ -19,6 +19,12 @@
   # Print deprecation notices to the Rails logger.
   config.active_support.deprecation = :log
 
+  # Use the lowest log level to ensure availability of diagnostic information
+  # when problems arise.
+  config.log_level = :debug
```

Comment:
Let's move this into config/application.rb and remove from the individual environment configurations.